shell/artifacts/Products/source/ProductFilter.js

```diff
@@ -37,7 +37,6 @@ defineParticle(({Particle}) => {
 
           let recipe = `
 ${this.serializeSchema(hostedParticle)}
-import '${hostedParticle.implFile.replace(/\.[^\.]+$/, '.manifest')}'
```

Comment:
Why don't we need this any more? I just want to make sure I am following correctly.

Reply:
This is an old particle implementation - then I used to add "import" statement to the inner recipe to import the hosted particle's manifest. However it doesn't always work because we don't know the full path.
Instead we now serialize the hosted particle spec (see line above: ${this.serializeSchema(hostedParticle)}).

This particle wasn't being used anywhere, so I added the serializeSchema when I created it, but forgot to remove the import statement.